c/tskit/tables.c

```diff
@@ -578,14 +582,31 @@ tsk_individual_table_get_row(
     return ret;
 }
 
+int
+tsk_individual_table_set_metadata_schema(
+    tsk_individual_table_t *self, const char *metadata_schema)
```

Comment:
I think we should change the signature to include a length, otherwise this will break with unicode inputs. Since we'll be allowing free text within the schema for description fields, I think we need to allow for arbitrary unicode.